csl-data.json

```diff
@@ -1,431 +1,440 @@
-            },
-            "event-place": {
-                "type": "string" 
-            },
-            "first-reference-note-number": {
-                "type": "string" 
-            },
-            "genre": {
-                "type": "string" 
-            },
-            "ISBN": {
-                "type": "string" 
-            },
-            "ISSN": {
-                "type": "string" 
-            },
-            "issue": {
-                "type": [
-                    "string",
-                    "number" 
-                ] 
-            },
-            "jurisdiction": {
-                "type": "string" 
-            },
-            "keyword": {
-                "type": "string" 
-            },
-            "locator": {
-                "type": "string" 
-            },
-            "medium": {
-                "type": "string" 
-            },
-            "note": {
-                "type": "string" 
-            },
-            "number": {
-                "type": [
-                    "string",
-                    "number" 
-                ] 
-            },
-            "number-of-pages": {
-                "type": "string" 
-            },
-            "number-of-volumes": {
-                "type": [
-                    "string",
-                    "number" 
-                ] 
-            },
-            "original-publisher": {
-                "type": "string" 
-            },
-            "original-publisher-place": {
-                "type": "string" 
-            },
-            "original-title": {
-                "type": "string" 
-            },
-            "page": {
-                "type": "string" 
-            },
-            "page-first": {
-                "type": "string" 
-            },
-            "PMCID": {
-                "type": "string" 
-            },
-            "PMID": {
-                "type": "string" 
-            },
-            "publisher": {
-                "type": "string" 
-            },
-            "publisher-place": {
-                "type": "string" 
-            },
-            "references": {
-                "type": "string" 
-            },
-            "reviewed-title": {
-                "type": "string" 
-            },
-            "scale": {
-                "type": "string" 
-            },
-            "section": {
-                "type": "string" 
-            },
-            "source": {
-                "type": "string" 
-            },
-            "status": {
-                "type": "string" 
-            },
-            "title": {
-                "type": "string" 
-            },
-            "title-short": {
-                "type": "string" 
+                ]
+            }
+        },
+        "numberDefinition": {
+            "oneOf": [
+                {
+                    "type": "number"
+                },
+                {
+                    "type": "string",
+                    "pattern": "^[0-9]+$"
```

Comment:
Changed this so that if it's a string, it is still just numbers in the string.

Reply:
I think that will break many things. `edition`, `issue`, `volume`often contain characters. I think it makes sense to save any functional changes for subsequent PRs.

Reply:
To my knowledge, this is the only potentially breaking change that I made. I'll revert this now and then we should be good to go @rmzelle.


@dhimmel feel free to give this another once-over if you'd like.